ignite/src/main/java/org/hibernate/ogm/datastore/ignite/impl/IgniteCacheInitializer.java

```diff
@@ -33,9 +43,20 @@
 
 	private static final long serialVersionUID = -8564869898957031491L;
 	private static final Log log = LoggerFactory.getLogger();
+	private static Map<Class<?>, Class<?>> h2TypeMapping;
+
+	static {
+		Map<Class<?>, Class<?>> map = new HashMap<>(  );
+		map.put( Character.class, String.class );
+
+		h2TypeMapping = Collections.unmodifiableMap( map );
+	}
+
+
```

Comment:
Could explain me what is it this h2TypeMapping?

Reply:
it is type changing.
returnValue = h2TypeMapping.getOrDefault( returnValue , returnValue  );

Reason: h2 not have field type 'character' and I use the changing for fix BuiltInTypeTest.testCharacterSupport()